src/components/AlgoGovernanceContent.jsx

```diff
@@ -83,7 +83,7 @@ export default function AlgoGovernanceContent() {
             setMaxBal(value.balance)
           }}
           // variant ={true}
-          disabled = {false}
+          disabled = {true}
```

Comment:
This should say value.committed === value.collateral instead of just being disabled by default

Reply:
People may add more collateral later if they so desire.